CMakeLists.txt

```diff
@@ -179,6 +179,10 @@ elseif (SUBSURFACE_TARGET_EXECUTABLE MATCHES "MobileExecutable")
 	endif()
 	list(APPEND QT_EXTRA_COMPONENTS QuickControls2)
 	add_definitions(-DSUBSURFACE_MOBILE)
+
+	# add definition to seperate mobile for devices and for desktop
+	add_definitions(-DSUBSURFACE_MOBILE_DESKTOP)
+
```

Comment:
Commenting on the commit messages for "/: add CMake definition SUBSURFACE_MOBILE_DESKTOP"...
I think this should be "Build system: add SUBSURFACE_MOBILE_DESKTOP as build type"

Reply:
Seems I misunderstood the guideline to use e.g. core: or mobile-widgets: when you make changes in that directory. Will update the commit messages.